extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -93,10 +94,7 @@
     <channel-type id="brightness">
         <item-type>Dimmer</item-type>
         <label>Brightness</label>
-        <description>
-            The brightness channel allows to control the brightness of a light.
-            It is also possible to switch the light on and off.
-        </description>
+        <description>Control the brightness and switch the light on and off.</description>
```

Comment:
no, I did not check other bindings. right now the formulations are in different forms. Lifx has something like `Sets the temperature of the light` while Hue has `The color temperature channel allows to set...`. I would tend not to refer to `channel` in the description but describe the capability/property of the thing it controls. wdyt?